packages/scripts/package.json

```diff
@@ -48,21 +48,26 @@
 		"clean-webpack-plugin": "^3.0.0",
 		"cross-spawn": "^5.1.0",
 		"css-loader": "^3.5.2",
+		"cwd": "^0.10.0",
 		"dir-glob": "^3.0.1",
 		"eslint": "^7.17.0",
 		"eslint-plugin-markdown": "^1.0.2",
+		"expect-puppeteer": "^4.4.0",
 		"file-loader": "^6.2.0",
 		"ignore-emit-webpack-plugin": "^2.0.6",
 		"jest": "^26.6.3",
-		"jest-puppeteer": "^4.4.0",
+		"jest-dev-server": "^4.4.0",
+		"jest-environment-node": "^26.6.2",
+		"jest-environment-puppeteer": "^4.4.0",
```

Comment:
Isn't this what we're providing in `packages/scripts/config/jest-environment-puppeteer/` now? :grimacing:

Reply:
Good catch! I think I did it in steps and replaced `jest-puppeteer` with `jest-environment-puppeteer` first. Removing it now.